packages/ui/src/components/Dropdown/dropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { Dropdown as DefaultDropdown, DropdownProps as DefaultDropdownProps } from 'antd'
+import { isFunction } from 'lodash'
```

Comment:
CHANGE IMPORT! 
`import funcName from 'lodash/funcName'` is decreasing bundle size